articles/dev-itpro/deployment/infrastructure-stack.md

```diff
@@ -32,7 +32,10 @@ ms.dyn365.ops.version: 8.1.1
 [!include[banner](../includes/banner.md)]
 [!include [banner](../includes/limited-availability.md)]
 
-Self-service deployment is available for Dynamics 365 for Finance and Operations cloud environments. Self-service deployment enables easier deployment and significantly reduced deployment times.
+Self-service deployment is available for Dynamics 365 for Finance and Operations **Tier 2 through 5** cloud environments. Self-service deployment enables easier deployment and significantly reduced deployment times.
+
+[!Important]
+We are rolling out this experience in waves. Currently, this experience is available only to a small number of NEW customers signing up for Finance & Operations. There is no change to the experience of our current customers having existing environments. 
```

Comment:
Remove currently and small number. People will ask how he small number of customers are getting selected. This experience will be made available in waves. This means that not all NEW customers signing up for Finance and Operations will see these changes.